includes/discovery/os/iosxe.inc.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
-if (str_contains($sysDescr, 'IOS-XE')) {
+if (str_contains($sysDescr, array('IOS-XE', 'X86_64_LINUX_IOSD'))) {
```

Comment:
We've not had reports that IOS-XE hasn't been working. Not sure we should be including this in until we do - are you seeing that sysDescr for IOS-XE based devices?